server/app/initializers/authority.rb

```diff
@@ -0,0 +1,12 @@
+require 'authority'
+
+Authority.configure do |config|
+  config.abilities =  {
```

Comment:
Apparently no :( Authority::UserAbilities reads those verbs and creates dynamically User.can_<verb>? methods. https://github.com/nathanl/authority/blob/master/lib/authority/user_abilities.rb#L11-L18